deploy/manifests/prod/us-east-2/tenant/storetheindex/indexstar/deployment.yaml

```diff
@@ -16,9 +16,6 @@ spec:
             - '--translateNonStreaming'
             # Use service names local to the namespace over HTTP to avoid
             # TLS handshake overhead.
-            - '--backends=http://dido-indexer:3000/'
-            - '--backends=http://kepa-indexer:3000/'
-            - '--backends=http://oden-indexer:3000/'
```

Comment:
do one at a time?

Reply:
It's going to be hard to see effect of disconnecting just one as the data is replicated. I will be keeping an eye on the metrics and revert if things start going wrong again.